### DISCUSSION TITLE
Dash for main character

### DISCUSSION THREAD

**Discussion by Arvi-beep-boop** [Quit975/DuckyGame#29] [Ideas]
Left - Right, Up-Down and Diagonal **dash** giving possibility to "teleport" 0.5f in specified direction

**Comment by ArcherWike** [comment]
🦆

**Comment by Quit975** [comment]
So teleport (instant) or dash(super fast but cintinuous)? What about enemies on the path? Ignored or still collides? Any cooldown? Sounds op without a cooldown

**Comment by Quit975** [comment]
I converted it into a discussion - it's a gameplay feature that has the possibility of greatly impacting the basic game loop. Definitely we don't want to experiment with it before having the solid prototype
